system/jlib/jflz.cpp

```diff
@@ -676,6 +682,9 @@ class jlib_decl CFastLZCompressor : public CInterface, public ICompressor
 
     virtual void open(void *buf,size32_t max)
     {
+        wrmax = max / 4;
+        if (wrmax < 256)
+            wrmax = max;
```

Comment:
Is this correct? e.g. if max = 1023.. max / 4 = 255, so wrmax = 1023 ?
Was it supposed to be something like wrmax = max>256?256:max on this line?